appinventor/blocklyeditor/src/blocks/procedures.js

```diff
@@ -902,3 +902,52 @@ Blockly.Blocks['procedures_callreturn'] = {
   typeblock: [{ translatedName: Blockly.Msg.LANG_PROCEDURES_CALLRETURN_TRANSLATED_NAME}]
 };
 
+Blockly.Blocks['map_list'] = {
```

Comment:
This block doesn't seem to quite work yet, so I would suggest commenting it out and making it work in a separate PR.  I also think that it should probably be in with the 'lists' category and in the lists.js file.

Reply:
Sounds good, I removed it for now, and I will make a separate PR in the future.